cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -417,7 +426,7 @@ private String rebalanceStateConditionType(KafkaRebalanceStatus status) {
             case Rebalancing:
                 return onRebalancing(reconciliation, host, apiClient, kafkaRebalance, rebalanceAnnotation);
             case Stopped:
-                return onStop(reconciliation, host, apiClient, kafkaRebalance, rebalanceAnnotation, rebalanceOptionsBuilder);
+                return onStop(kafkaRebalance, reconciliation, host, apiClient, rebalanceAnnotation, rebalanceOptionsBuilder);
```

Comment:
As I have said before, can you keep the method signatures (argument ordering) consistent. `reconciliation` should be first with `kafkaRebalance` after `apiClient`.

Reply:
Iam sorry, I guess it got skipped. I had resolved the ordering earlier. I have resolved it now